mansoursaid001/CSEN1059-web
# Admin create users

@MariamMohamedFawzy @silvermist @mansoursaid001 
Can you please review this?

To test:
0) make sure to run 'php artisan migrate:refresh --seed' to run the seeds
1) log in as an admin
2) from the side bar click on admin panel > support supervisors or support agents or support agents
3) try to create a new user and put your actual email
4) check that the user is created and that the email is sent..also check that the user can be seen at the bottom of the page
5) this panel is only visible to the admin and you can also check that.

thank you (Y)
